client/my-sites/plugins/plugins-list/style.scss

```diff
@@ -1,56 +1,25 @@
-.plugins-list {
-	margin-bottom: 16px;
-}
-
-.plugins-list .plugin-item {
-	@include breakpoint-deprecated( "<1040px" ) {
-		width: 100%;
-
-		&:nth-last-child(n + 2) {
-			border-bottom-width: 1px;
-		}
-	}
-
-	@include breakpoint-deprecated( ">1040px" ) {
-		flex-direction: column;
-		width: 33.33%;
-		border-right-width: 1px;
-
-		&:nth-last-child(n + 4) {
-			border-bottom-width: 1px;
-		}
-
-		// Provider proper bottom borders in the second-last row when the number
-		// of items is not multiple of 3.
-		&:nth-child(3n) {
-			border-right-width: 0;
-			&:nth-last-child(3),
-			&:nth-last-child(2) {
-				border-bottom-width: 1px;
-			}
-		}
-
-		&:nth-child(3n - 1):nth-last-child(3) {
-			border-bottom-width: 1px;
-		}
-
-		.plugin-item__count,
-		.plugin-item__actions {
-			align-self: flex-end;
-			flex-direction: column;
-			margin-top: -6px;
-			padding-top: 0;
-			text-align: right;
-		}
+body.is-section-plugins .plugin-management-wrapper {
+    --wp-components-color-accent: var(--studio-automattic-blue-50);
 
-		.plugin-item__count {
-			flex-direction: row;
-		}
+	.navigation-header {
+		border-block-end: 1px solid var(--color-border-secondary);
+		padding-inline: 48px;
 	}
 }
 
-.plugins-list__elements {
-	display: flex;
-	flex-flow: row wrap;
-	padding: 0;
+body.is-section-plugins .plugin-management-wrapper,
+.is-section-jetpack-cloud-plugin-management {
+
+    .dataviews-wrapper .dataviews-view-table {
+        .dataviews-view-table__row {
+            .plugin-icon {
+                max-width: 35px;
+                max-height: 35px;
+            }
+
+            .dataviews-view-table-selection-checkbox{
+                padding-left: 0;
```

Comment:
Same as above, these overrides shouldn't exist in the first place:

https://github.com/Automattic/wp-calypso/blob/trunk/client/components/dataviews/style.scss#L21

IMHO we should try to clean them up and get the design as close as possible to the core experience. This can be good feedback for the core package, too.

Reply:
Good suggestion, will follow-up [in this task](https://github.com/Automattic/dotcom-forge/issues/9554).

Reply:
If we use DataViews directly, would these issues exist? Other way to put it: are they coming from one of Calypso's components?

In any case, I'd like to suggest centralizing all overrides in a single file https://github.com/Automattic/wp-calypso/blob/trunk/client/components/dataviews/style.scss#L204 Every one of them should have a PR or issue linked, so it's easy to identify and remove them. We shouldn't remove the existing ones, not introducing more.

Reply:
Even after the [DataViews directly usage commit](https://github.com/Automattic/wp-calypso/pull/95561/commits/8547dfac0a99aa25e9db3233656fceb672dcfeea), we still get this problem. But it's not a core component issue and indeed a custom one [we introduced here](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/dataviews/style.scss#L21).

I updated [the follow-up task](https://github.com/Automattic/dotcom-forge/issues/9554) to reflect that this is an internal change improvement.

Reply:
Thanks for investigating. Can we centralize this hotfix [in file](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/dataviews/style.scss#L204), scoping it to this screen, and linking to the follow-up task? I feel that'd be a better way to collaborate among all the people working in calypso/dataviews.

Reply:
Sounds good, [moved it here](https://github.com/Automattic/wp-calypso/pull/95561/commits/008155def6b6b8549e16a6a0a797a65d4020a031), and added [the follow-up task in the css](https://github.com/Automattic/wp-calypso/pull/95561/commits/5dae73ac2845435222767aeb5b0683fcb6b1990f).